package.json

```diff
@@ -29,10 +29,15 @@
     "node": "^6 || ^7 || ^8 || ^9 || ^10 || ^11 || ^12 || >=13.7"
   },
   "bin": "./cli.js",
+  "typings": "./index.d.ts",
```

Comment:
Can you rename to `types` to keep consistency across my projects?

Reply:
Sure